SPECS/pcsc-lite/pcsc-lite.spec

```
@@ -1,16 +1,13 @@
-Name:           pcsc-lite
-Version:        1.9.0
-Release:        2%{?dist}
 Summary:        PC/SC Lite smart card framework and applications
-
+Name:           pcsc-lite
+Version:        1.9.5
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://pcsclite.apdu.fr/
 Source0:        https://pcsclite.apdu.fr/files/%{name}-%{version}.tar.bz2
-Source1:        https://pcsclite.apdu.fr/files/%{name}-%{version}.tar.bz2.asc
-Source2:	gpgkey-F5E11B9FFE911146F41D953D78A1B4DFE8F9C57E.gpg
-Source3:	org.debian.pcsc-lite.policy
+Source1:	      org.debian.pcsc-lite.policy
```

I think accidentally introduced a tab here now.
```suggestion
Source1:        org.debian.pcsc-lite.policy
```